packages/hoppscotch-app/src/components/environments/index.vue

```diff
@@ -155,9 +155,9 @@
         @update-selected-team="updateSelectedTeam"
       />
     </div>
-    <EnvironmentsMy v-if="environmentType.type === 'my-environments'" />
+    <EnvironmentsMy v-show="environmentType.type === 'my-environments'" />
     <EnvironmentsTeams
-      v-else
+      v-show="environmentType.type === 'team-environments'"
```

Comment:
Why are you changing this from `v-if` to `v-show` ?

Reply:
@AndrewBastin so the modals can be called even when the correspondent tab is not active;

Reply:
Is there any case where not using a `v-show` causes an issue ? Because I still don't exactly understand why it is needed, not rendering the modal components if not required is better than v-show for this case, unless we are using some mounted state stuff.

It would be awesome if you could provide a steps to repro for a condition which breaks when you use v-show.

Reply:
@AndrewBastin To reproduce you can select the tab for "team environment" and if you try to edit some global variable the modal is not shown, because `EnvironmentsMy` where the modal is originally from is not in the dom with `v-if`

Reply:
Yeah, okay, makes sense. I am not a fan of Global being hidden in Teams though as they are being shared, but that is not in scope for this PR, hence, I will allow it.